### ISSUE TITLE
Consider adding instructions on how to increase boot2docker memory limit

### ISSUE THREAD

**Issue by ghost** [VISIONAI/clouddream#25]
I'm familiar with docker, but not boot2docker on OSX, and it took me quite a while to figure out how to increase the VM's memory size. You reference that it might be necessary to up the RAM for processing of larger images, so it would be helpful to have a quick link on how to change the configuration to match.

I followed the instructions here: https://docs.docker.com/articles/b2d_volume_resize/

Specifically,
1. Initialise a default file to customize 
   `boot2docker config > ~/.boot2docker/profile command.`
2. Edit the file and change from 2 Gig to 6 Gig
   `Memory = 2048`
   to
   `Memory = 6144`
3. Run the following commands:

```
$ boot2docker poweroff
$ boot2docker destroy
$ boot2docker init
$ boot2docker up
```

**Comment by jontonsoup** [comment]
@ihaventkilledanybodysince1984 did this fix your memory issues? 

I'm having memory issues after increasing my VM to 10g. It seems like I cant make images over 1000 px.

**Comment by ghost** [comment]
Yeah, it fixed my issues. I think I upped mine to 8G, and could do 1600X1800 pictures, IIRC.
